app/src/main/java/org/oppia/android/app/testing/AdministratorControlsFragmentTestActivity.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.android.app.testing
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.ActivityComponentImpl
+import org.oppia.android.app.administratorcontrols.LogoutDialogFragment
+import org.oppia.android.app.administratorcontrols.RouteToAppVersionListener
+import org.oppia.android.app.administratorcontrols.RouteToProfileListListener
+import org.oppia.android.app.administratorcontrols.ShowLogoutDialogListener
+import org.oppia.android.app.administratorcontrols.appversion.AppVersionActivity
+import org.oppia.android.app.drawer.NAVIGATION_PROFILE_ID_ARGUMENT_KEY
+import org.oppia.android.app.settings.profile.ProfileListActivity
+import org.oppia.android.app.testing.activity.TestActivity
+import org.oppia.android.app.translation.AppLanguageResourceHandler
+import javax.inject.Inject
+
+/** Activity [AdministratorControlsFragmentTestActivity] that allows user to change admin controls. */
```

Comment:
```suggestion
/** Test activity for ``AdministratorControlsFragmentTestActivity``. */
```

I suspect this might be a bit clearer.